domains/measurements/Measurements.API/API/MeteringPoints/Api/v2024_01_10/Dto/Responses/Address.cs

```diff
@@ -0,0 +1,21 @@
+namespace API.MeteringPoints.Api.v2024_01_10.Dto.Responses;
+
+public class Address
```

Comment:
Why not a record here like the rest?

Reply:
Dunno. This is taken from DataSync. I can make it a record if you want?

Reply:
Made record